hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/BackgroundPipelineCreator.java

```diff
@@ -64,7 +65,8 @@ public class BackgroundPipelineCreator implements SCMService {
    * SCMService related variables.
    * 1) after leaving safe mode, BackgroundPipelineCreator needs to
    *    wait for a while before really take effect.
-   * 2) NewNodeHandler, NonHealthyToHealthyNodeHandler, PreCheckComplete
+   * 2) NewNodeHandler, NodeIpOrHostnameUpdateHandler,
```

Comment:
Shouldn't this be "NodeAddressUpdateHandler" instead of "NodeIpOrHostnameUpdateHandler"?

Reply:
Good catch. Let me fix this.